enhancements/installer/assisted-installer-bare-metal-validations.md

```diff
@@ -0,0 +1,280 @@
+---
+title: assisted-installer-bare-metal-validations
+authors:
+  - "@dhellmann"
+  - "@stbenjam"
+reviewers:
+  - "@romfreiman"
+  - "@yevgeny-shnaidman"
+approvers:
+  - "@hardys"
+  - "@crawford"
+  - "@markmc"
+creation-date: 2020-06-24
+last-updated: 2020-06-26
+status: implementable
+---
+
+# Assisted Installer Bare Metal Validations
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The assisted installer supports a user-provisioned infrastructure
+installation workflow for bare-metal clusters, but configures the
+installed cluster with certain capabilities that are currently only
+available using the `baremetal` platform in an installer-provisioned
+infrastructure workflow.
+
+This enhancement describes how we can modify the installer to allow
+bare-metal user-provisioned infrastructure clusters, including those
+produced with the assisted installer, to include those
+installer-provisioned infrastructure capabilities.
+
+## Motivation
+
+The [connected-assisted-installer
+enhancement](./connected-assisted-installer.md) describes how an agent
+running on several bare-metal hosts receive a command from the
+assisted installer application to begin the installation of that
```

Comment:
Personally, I dont like the term application. I would call it either assisted installer or assisted installer service